metaschema-cli/src/main/java/gov/nist/secauto/metaschema/cli/commands/AbstractValidateContentCommand.java

```diff
@@ -116,38 +117,6 @@ public List<ExtraArgument> getExtraArguments() {
   @SuppressWarnings("PMD.PreserveStackTrace") // intended
   @Override
   public void validateOptions(CallingContext callingContext, CommandLine cmdLine) throws InvalidArgumentException {
-    if (cmdLine.hasOption(CONSTRAINTS_OPTION)) {
-      String[] args = cmdLine.getOptionValues(CONSTRAINTS_OPTION);
-      for (String arg : args) {
-        Path constraint = Paths.get(arg);
-        if (!Files.exists(constraint)) {
-          throw new InvalidArgumentException(
-              "The provided external constraint file '" + constraint + "' does not exist.");
-        }
-        if (!Files.isRegularFile(constraint)) {
-          throw new InvalidArgumentException(
-              "The provided external constraint file '" + constraint + "' is not a file.");
-        }
-        if (!Files.isReadable(constraint)) {
-          throw new InvalidArgumentException(
-              "The provided external constraint file '" + constraint + "' is not readable.");
-        }
-      }
-    }
-
-    List<String> extraArgs = cmdLine.getArgList();
-    if (extraArgs.size() != 1) {
-      throw new InvalidArgumentException("The source to validate must be provided.");
-    }
```

Comment:
Since this isn't a file check, should I restore this.

Reply:
Yes. Please put this back.

Reply:
Fixed in https://github.com/usnistgov/metaschema-java/pull/298/commits/e51fc11ed999007ea6b42683398b62fbe287d031.